modules/private.php

```diff
@@ -0,0 +1,307 @@
+<?php
```

Comment:
Since this file is brand new, could you add it (and the `modules/private` directory) to the PHPCS whitelist here?
https://github.com/Automattic/jetpack/blob/08a66d779ceb27fc5c98324ec4df8a0fa9f68f6f/bin/phpcs-whitelist.js#L12

You may then have to fix a few PHPCS settings in that file.

Reply:
https://github.com/Automattic/jetpack/commit/b2913ad21aff4256b1650bab98106bd82bfc5ca7 did all of that; not sure what happened to it.

Reply:
Strange, seems to have been dropped. Nevertheless, I've merged in the changes from your commit to [6326e10](https://github.com/Automattic/jetpack/pull/10333/commits/6326e102e91c2fe90413a43b13d16a04e3a65cf5)

Reply:
Same, marking as resolved but the linked commit was dropped.